automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -186,6 +186,10 @@ class RuntimeCreationDiskSpec
         )
       )
 
+      val createRuntime2Request = createRuntimeRequest.copy(toolDockerImage =
+        Some(ContainerImage(LeonardoConfig.Leonardo.pythonImageUrl, ContainerRegistry.GCR))
+      ) //this just needs to be a different image from default image Leonardo uses, which is gatk
```

Comment:
is there a reason this needs to be a difference image?

Reply:
this is the verify that using the same PD, it still works if we create a new runtime with a different image

Reply:
ah makes sense!